templates/_newsletter.njk

```diff
@@ -3,18 +3,29 @@
 {% set thispage = 'newsletters' %}
 {% set pagedesc = pagedesc if pagedesc else 'Newsletter: Manga and Anime Society Kharagpur' %}
 
-{% macro nextprev() %}
+{% set pagecount = pages.length %}
+
+
+
+{% macro nextprevnewsletter() %}
 	<div class="controls">
-		<a{% if adjs[0] %} href="/newsletters/{{ adjs[0] }}"{% endif %}><button class="rel" {{ 'disabled' if not adjs[0] }}><span class="buttonicon" id="flipped">&#x279C;</span><span class="buttontext">Previous</span></button></a>
+		<a{% if adjs[0] %} href="/newsletters/{{ adjs[0] }}"{% endif %}><button class="rel" {{ 'disabled' if not adjs[0] }}><span class="buttonicon">&#x21B0;</span><span class="buttontext">Prev Newsletter</span></button></a>
```

Comment:
I was hoping to change the wordy "prev/next newsletter' to some icon or something, 
but for now its ok